20200416/cli.py

```diff
@@ -55,7 +55,13 @@ def get_parser():
     subparsers = parser.add_subparsers(help="sub-command help")
 
     # TODO: Add `check-level subparser
+    parser_check = subparsers.add_parser('check-level', help='check log level')
+    parser_check.add_argument("{debug,info,warning,error}", help="the level of log message")

```

Comment:
```suggestion
    parser_check.add_argument(
        "level",
        type=str,
        choices=["debug", "info", "warning", "error"],
        help="the level of log message"
    )
    parser_check_level.set_defaults(func=check_level)

```

이렇게 postional argument 값이 설정이 되고,
함수를 연동하시면 됩니다.